src/main/java/org/testng/IInvokedMethodListener2.java

```diff
@@ -7,16 +7,19 @@
  */
 public interface IInvokedMethodListener2 extends IInvokedMethodListener {
```

Comment:
Please have the interface contents moved into its parent and have this interface deprecated.